geopaparazzispatialitelibrary/src/eu/geopaparazzi/spatialite/database/spatial/util/SpatialiteUtilities.java

```diff
@@ -43,12 +43,74 @@
      * Array of fields that will be ingored in attributes handling.
      */
     public static String[] IGNORED_FIELDS = {SPATIALTABLE_ID_FIELD, "PK_UID", "_id"};
-
+     
     /**
      * Name/path separator for spatialite table names.
      */
     public static final String UNIQUENAME_SEPARATOR = "#"; //$NON-NLS-1$
 
+    /**
+     * Active Connection to a  Spatialite Database
+    * <p/>
+     * <p>Indented for general use for Transformation etc.
+     */
+    private static Database dbSpatialite=null;
+    /**
+     * Active Connection to a  Spatialite Database
+    * <p/>
+     * <p>Indented for general use for Transformation etc.
+     */
+    private static Database dbMemSpatialite=null;
```

Comment:
2015-09-19 14:30 GMT+02:00 Andrea Antonello <notifications@github.com>:

> In
> geopaparazzispatialitelibrary/src/eu/geopaparazzi/spatialite/database/spatial/util/SpatialiteUtilities.java
> <https://github.com/geopaparazzi/geopaparazzi/pull/287#discussion_r39919170>
> :
>
> >      /**
> >       * Name/path separator for spatialite table names.
> >       */
> >      public static final String UNIQUENAME_SEPARATOR = "#"; //$NON-NLS-1$
> >
> > +    /**
> > +     * Active Connection to a  Spatialite Database
> > +    * <p/>
> > +     * <p>Indented for general use for Transformation etc.
> > +     */
> > +    private static Database dbSpatialite=null;
> > +    /**
> > +     * Active Connection to a  Spatialite Database
> > +    * <p/>
> > +     * <p>Indented for general use for Transformation etc.
> > +     */
> > +    private static Database dbMemSpatialite=null;
>
> Do we really want to support memory databases? I can't find a good use for
> that on android. Also because the OS might kill your app while you are
> logging due to memory issues.
> Should be better removed?
>
This was intended for the case where no active spatialite connection exists
* transforming the position

The static one would be set by the first read spatialite connection
* a memory only if none exist

was the idea.

I also don't like the idea.

If the Database used in 'GPApplication'
* used a spatialite connection (jsqlite.Database)
   * then that could be used

> —
> Reply to this email directly or view it on GitHub
> <https://github.com/geopaparazzi/geopaparazzi/pull/287/files#r39919170>.
>